src/main/java/org/jboss/tools/intellij/openshift/utils/odo/Odo.java

```diff
@@ -11,110 +11,83 @@
+  ComponentInfo getComponentInfo(String project, String component, String path, ComponentKind kind) throws IOException;
 
-    void deleteProject(String project) throws IOException;
+  void deleteComponent(String project, String context, String component, ComponentKind kind) throws IOException;
 
-    void setProject(String project) throws IOException;
+  boolean isLogRunning(String component, boolean deploy);
 
-    void login(String url, String userName, char[] password, char[] token) throws IOException;
+  void createProject(String project) throws IOException;
 
-    boolean isAuthorized();
+  void deleteProject(String project) throws IOException;
 
-    List<Component> getComponents(String project) throws IOException;
+  void setProject(String project) throws IOException;
 
-    List<Service> getServices(String project) throws IOException;
+  void login(String url, String userName, char[] password, char[] token) throws IOException;
 
-    void about() throws IOException;
+  boolean isAuthorized();
 
-    Binding link(String context, String target) throws IOException;
+  List<Service> getServices(String project) throws IOException;
 
-    List<Binding> listBindings(String context) throws IOException;
+  void about() throws IOException;
 
-    void deleteBinding(String context, String binding) throws IOException;
+  Binding link(String context, String target) throws IOException;
 
-    String consoleURL() throws IOException;
+  List<Binding> listBindings(String context) throws IOException;
 
-    void debug(String context, Integer port) throws IOException;
+  void deleteBinding(String context, String binding) throws IOException;
 
-    DebugStatus debugStatus(String context) throws IOException;
+  String consoleURL() throws IOException;
 
-    java.net.URL getMasterUrl();
+  void debug(String context, Integer port) throws IOException;
 
-    List<ComponentDescriptor> discover(String path) throws IOException;
+  java.net.URL getMasterUrl();
 
-    ComponentTypeInfo getComponentTypeInfo(String componentType, String registryName) throws IOException;
+  List<ComponentDescriptor> discover(String path) throws IOException;
 
-    List<DevfileRegistry> listDevfileRegistries() throws IOException;
+  ComponentTypeInfo getComponentTypeInfo(String componentType, String registryName) throws IOException;
 
-    void createDevfileRegistry(String name, String url, String token) throws IOException;
+  List<DevfileRegistry> listDevfileRegistries() throws IOException;
 
-    void deleteDevfileRegistry(String name) throws IOException;
+  void createDevfileRegistry(String name, String url, String token) throws IOException;
 
-    List<DevfileComponentType> getComponentTypes(String name) throws IOException;
+  void deleteDevfileRegistry(String name) throws IOException;
 
-    boolean isOpenShift();
+  boolean isOpenShift();
 
-    void migrateComponent(String name);
+  void migrateComponent(String name);
 
+  boolean checkPodman() throws IOException;
```

Comment:
Wouldn't a more speaking name be `canExecutePodman()`? Unless I am misreading the purpose of `OdoCli.checkPodman()`

Reply:
good review. i'll change that.